app/controllers/wiki_controller.rb

```diff
@@ -353,7 +353,7 @@ def stale
       .references(:node_revisions)
       .group('node_revisions.nid, node_revisions.vid')
       .order('node_revisions.timestamp ASC')
-      .where("node_revisions.status = 1 AND node.status = 1 AND type = 'page'"))
+      .where("node_revisions.status = 1 AND node.status = 1 AND (type = 'page' OR type = 'tool' OR type = 'place')"), items: 10)
```

Comment:
See comment above on `type = tool OR type = place`

Reply:
hey @RuthNjeri, yeah I think my version of code was old and hence github got confused. I only added the "items: 10" part. I will make the required changes. Thank you so much

Reply:
@RuthNjeri You can check now